spring-integration-core/src/main/java/org/springframework/integration/dispatcher/UnicastingDispatcher.java

```diff
@@ -145,7 +145,8 @@ private boolean doDispatch(Message<?> message) {
 				success = true; // we have a winner.
 			}
 			catch (Exception e) {
-				RuntimeException runtimeException = this.wrapExceptionIfNecessary(message, e);
+				@SuppressWarnings("deprecation")
+				RuntimeException runtimeException = wrapExceptionIfNecessary(message, e);
```

Comment:
Why don't use a new utility here?
Is there some breaking changes in the new utility method?
Thanks

Reply:
Yes; the utility always wraps in `MessageDeliveryException` see the comment in `AbstractDispatcher`; need to wait until 5.1 to switch to the utility.